debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -338,6 +348,11 @@ public SqlServerConnectorConfig(Configuration config) {
 
         this.databaseName = config.getString(DATABASE_NAME);
         this.snapshotMode = SnapshotMode.parse(config.getString(SNAPSHOT_MODE), SNAPSHOT_MODE.defaultValueAsString());
+
+        if (snapshotMode == SnapshotMode.INITIAL_SCHEMA_ONLY) {
```

Comment:
@gunnarmorling I must admit I am not sure about the purpose of this code

Reply:
LOL, that's a left-over. Sorry. It's removed now.